eventuate-tram-messaging/src/test/resources/default.dialect.properties

```diff
@@ -0,0 +1,2 @@
+eventuate.current.time.in.milliseconds.sql=some custom sql
```

Comment:
You can replace all this test dialect.properties files with property definitions in the test `@SpringBootTest(properties = {"foo=bar", "bar=foo"})`

Reply:
ok